Electron/src/pages/StartMenu/RegisterMenu.tsx

```diff
@@ -130,6 +145,11 @@ export default function RegisterMenu({ setIsSigningUp }: PropsRegisterMenu) {
 
   return (
     <div className={`${styles.mainModalContainer}`}>
+      {loading && (
```

Comment:
I found `LoadingCircleSmallNoPadding` component being used in the song uploading button. However, in order to have a proper spacing in the login and register button, I'll need to modify the `LoadingCircleSmallNoPadding` component a bit by removing the wrapping div, because the div causes the loading component to take up too much width inside the button, therefore, causing the login text to wrap down to 2nd line.

this is how the change would affect the song upload button:

**before**
<img width="405" alt="image" src="https://github.com/user-attachments/assets/72c93839-d98a-405b-a008-67542c678e0b">

**after**
<img width="426" alt="image" src="https://github.com/user-attachments/assets/2eaa2d01-2acd-4527-8226-ecc55f5db298">

and for login button, the loading will look like this after modifying the component:
<img width="393" alt="image" src="https://github.com/user-attachments/assets/c1654cc7-0fbf-4d59-8060-bf5d6b2ecb90">

Reply:
Hi @gigigimay , I'm fine with it. We will have to check it doesn't break other components :)

Reply:
Hi, from the current code base, I only see the component being used once inside the song upload button, and the impact on it can be seen in the screenshots in the above comment. So I believe it should not break anything else. ;)

Reply:
Perfect @gigigimay , thanks for taking a look at possible impacts :)